game-engine/src/jvmTest/kotlin/za/co/entelect/challenge/game/engine/renderer/WormsGameDetailsTest.kt

```diff
@@ -0,0 +1,47 @@
+package za.co.entelect.challenge.game.delegate.renderer
+
+import za.co.entelect.challenge.game.delegate.factory.TEST_CONFIG
+import za.co.entelect.challenge.game.engine.command.CommandStrings
+import za.co.entelect.challenge.game.engine.command.feedback.StandardCommandFeedback
+import za.co.entelect.challenge.game.engine.config.GameConfig
+import za.co.entelect.challenge.game.engine.factory.TestMapFactory.buildMapWithCellType
+import za.co.entelect.challenge.game.engine.factory.TestWormsPlayerFactory.buildWormsPlayerDefault
+import za.co.entelect.challenge.game.engine.factory.TestWormsPlayerFactory.buildWormsPlayers
+import za.co.entelect.challenge.game.engine.map.CellType
+import za.co.entelect.challenge.game.engine.map.Point
+import za.co.entelect.challenge.game.engine.player.WormsPlayer
+import za.co.entelect.challenge.game.engine.renderer.WormsGameDetails
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class WormsGameDetailsTest {
+
+    private val config: GameConfig = TEST_CONFIG
+
+    @Test
+    fun test_single_last_move_added_correctly() {
+        val wormsPlayers = buildWormsPlayerDefault(config)
+        wormsPlayers.forEachIndexed { i, p ->
```

Comment:
take a look at `TestWormsPlayerFactory.kt`, it can probably create what you require here

Reply:
This is how it was done in `WormsRendererTest.kt`.  I'll check whether the factory has something more convenient though.  If it doesn't then refactoring that, I think, falls outside the scope of this change(?)

Reply:
So it turns out that the other factory method for creating worms also doesn't initialise the worm positions.  I'm going to go ahead and leave it as is.  WDYT?

Reply:
nah this is good then, i was just concerned with the positions being set by index value, and then a move command is created later which needs to be spatially close to these, or else the command validation could fail
So this is fine since it does fully test this property